modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -76,6 +76,14 @@ export default {
         return 20;
       }
     },
+    tooltip () {
+      let msg = false;
+      if (this.field.readOnly) {
+        // TODO For editor-facing tooltips is this the correct language?
```

Comment:
Not sure if there should be any more nuance to the generic tooltip message. Disabled? Read Only?

Reply:
🤷‍♂️